db/migrations/0003_alter_player_email.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2023-10-24 21:33
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_alter_player_created_at'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='player',
+            name='email',
+            field=models.EmailField(max_length=255),
+        ),
+    ]
```

Comment:
Include one migration per app in your pull request. It is a good practice for Django projects to have all changes made to the models in one migration per app per PR.